libraries/shared/factories/event/README.md

```diff
@@ -23,52 +23,41 @@ The event watcher is responsible for continuously fetching and delegating chunks
 
 Using the `compose` or `composeAndExecute` command, event watchers can be loaded with plugin event transformers and execute over them.
 
-### [Event Transformer](../staging/libraries/shared/transformer/event_transformer.go)
+### [Configured Transformer](../staging/libraries/shared/factories/event/transformer.go)
```

Comment:
Ah, seems relevant to one of my comments on #52

Reply:
Taking a look at this - it appears the `staging` is in the links so they work on github and explicitly point at the staging branch. However that means they don't work locally.

The link still seems wrong because you need to go back further than one directory. For event watcher you need to be `../../../../staging/libraries/shared/watcher/event_watcher.go` and for the configured transformer `../../../../staging/libraries/shared/factories/event/transformer.go`.  If I counted my dots right that should mean you are at the right spot, on the staging branch. The other alternative is to be at the "current" branch, in which case you don't have to go back as far and can skip naming staging entirely.